.vscode/launch.json

```diff
@@ -11,7 +11,7 @@
             "skipFiles": [
                 "<node_internals>/**"
             ],
-            "program": "${workspaceFolder}\\build\\build.js",
+            "program": "${workspaceFolder}/build/build.js",
```

Comment:
This is required for this to work on linux or osx, it should still function on windows as node.js fs operations support unix style paths on windows.